src/main/java/io/confluent/connect/jdbc/dialect/GenericDatabaseDialect.java

```diff
@@ -152,6 +156,7 @@ public DatabaseDialect create(AbstractConfig config) {
   private volatile JdbcDriverInfo jdbcDriverInfo;
   private final int batchMaxRows;
   private final TimeZone timeZone;
+  private final String tsGranularity;
```

Comment:
Any reason this is a string instead of an enum? See `NumericMapping` for an example of using enum in config.